### PR TITLE
Spanish welcome guide

### PR DESCRIPTION
**Pull request by Graywolf9**
Added a document that works as a welcome and contribution guide to spanish users

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ subdirectories:
 - [Russian translation guide / Участие в переводе](ru/translation-guide.md)
 - [Simplified Chinese Guide / 简体中文翻译指南](zh-cn/translation-guide.md)
 - [Traditional Chinese Guide / 正體中文翻譯指南](zh-tw/translation-guide.md)
+- [Spanish guide / Guía en español](es/README.md)
```
Comment (JuanVqz) [inline docs/README.md:12]: I like the idea of using `README.md` but seeing other teams it seems like the standard is `translation-guide.md` and we may be sticky to that convention

**Review by JuanVqz — COMMENTED** [review]
:100: Gracias @Graywolf9

**Review by JuanVqz — APPROVED** [review]
Let's do it!

**Comment by marcelozarate** [comment]
Aaaahhh no me dió tiempo a aportar, ahora voy a tener que abrir un PR 👍

**Review by marcelozarate — COMMENTED** [review]
👌

**Pull request merged**